client/src/pages/demo/components/index.tsx

```diff
@@ -32,6 +33,23 @@ const DemoComponent: React.FunctionComponent = () => {
         <br />
         <hr className="w-3/4 mx-auto" />
       </div>
+
+      <div className="p-4 border-b border-black" id='dropdown'>
```

Comment:
- as per standard. use double quotes for html tags
```suggestion
      <div className="p-4 border-b border-black" id="dropdown">
```